test/sim/team-validator/breeding.js

```diff
@@ -203,17 +203,22 @@ describe('Team Validator', function () {
 		assert.legalTeam(team, 'gen3ou');
 	});
 
-	it.skip('should reject Volbeat with both Lunge and Dizzy Punch in Gen 7', function () {
+	it('should allow complex chainbred sets', function () {
 		team = [
-			{species: 'volbeat', ability: 'swarm', moves: ['lunge', 'dizzypunch'], evs: {hp: 1}},
+			{species: 'toxicroak', ability: 'dryskin', moves: ['bulletpunch', 'crosschop', 'fakeout'], evs: {hp: 4}},
 		];
-		assert.false.legalTeam(team, 'gen7anythinggoes');
+		assert.legalTeam(team, 'gen5ou');
+
+		team = [
+			{species: 'corphish', ability: 'hypercutter', moves: ['dragondance', 'metalclaw'], evs: {hp: 1}},
+		];
+		assert.legalTeam(team, 'gen4ou');
 	});
 
-	it.skip('should accept this chainbreed on Toxicroak', function () {
+	it.skip('should reject Volbeat with both Lunge and Dizzy Punch in Gen 7', function () {
```

Comment:
should this skip be removed?

Reply:
This Volbeat still passes as legal when it shouldn't, and I don't think a fix for this has been coded yet.

Reply:
This is just the diff being weird and you didn't actually touch the Volbeat test, right?

Reply:
Yeah, I moved up the Toxicroak test to separate it from the skipped test.